code/game/objects/items/weapons/batons.dm

```diff
@@ -108,7 +112,9 @@
   * * user - The attacking user
   */
 /obj/item/melee/classic_baton/proc/on_non_silicon_stun(mob/living/target, mob/living/user)
-	return
+	var/armour = target.run_armor_check("chest", armour_penetration = stamina_armour_pen)
+	armour = (100 - armour) / 100
+	target.adjustStaminaLoss(stamina_damage * armour)
```

Comment:
```suggestion
	var/factor = (100 - armour) / 100
	target.adjustStaminaLoss(stamina_damage * factor)
```
or 
```suggestion
	var/damage = stamina_damage * (100 - armour) / 100
	target.adjustStaminaLoss(damage)
```
or even just
```suggestion
	target.adjustStaminaLoss(stamina_damage * (100 - armour) / 100)
```

Inverting the semantics of a variable midproc ("armor" becoming "penetration") is very confusing to read.

Reply:
added went with the first one, and added some bonus comments